portal-api/api-tests/v1/gef/files.api-test.js

```diff
@@ -114,15 +112,14 @@ describe(baseUrl, () => {
         },
       ];
 
-      const { status, body } = await as(aMaker)
+      const { body, status } = await as(aMaker)
         .postMultipartForm({ parentId: mockDeal.body._id }, invalidFiles)
         .to(baseUrl);
 
-      expect(status).toEqual(200);
+      expect(status).toEqual(400);
```

Comment:
Slightly changed behaviour so that if files fail the multer filter we fail fast and return a 400 before trying to upload them to the server, failing and returning a 200 with no files uploaded. I think this makes more sense and follows the 'fail fast' principle but happy to discuss